### PR TITLE
Fix cosmos pipeline

### PR DESCRIPTION
**Pull request by Netyyyy**
# Description
Fix cosmos pipeline

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/spring/pipeline/cosmos-integration-matrix-for-cosmos.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (saragluna) [inline sdk/spring/pipeline/cosmos-integration-matrix-for-cosmos.json:1]: why don't we still call it sdk/spring/pipeline/cosmos-integration-matrix.json?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/pipeline/cosmos-integration-matrix-for-cosmos.json (hunk 1 above)
Comment (Netyyyy) [reply sdk/spring/pipeline/cosmos-integration-matrix-for-cosmos.json:1]: because spring use the old one to test multiple versions

**Review by JimSuplizio — APPROVED** [review]
My approval is only for the removal of sdk/cosmos/ci.spring.yml which is only testing azure-spring-data-cosmos. The reason this yml file needed to be removed is because this was already a pipeline, [sdk/spring/ci.cosmos.yml](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/ci.cosmos.yml#L33), testing azure-spring-data-cosmos.

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @Netyyyy and @saragluna

**Review by saragluna — APPROVED** [review]

**Comment by Netyyyy** [comment]
Hi @alzimmermsft @srnagar @g2vinay @conniey, could one of you help approve the changes below '.vscode'?

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**